GrammarNazi.Core/BotCommands/Telegram/BaseTelegramCommand.cs

```diff
@@ -0,0 +1,85 @@
+﻿using GrammarNazi.Core.Extensions;
+using System;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Telegram.Bot;
+using Telegram.Bot.Types;
+using Telegram.Bot.Types.Enums;
+using Telegram.Bot.Types.ReplyMarkups;
+
+namespace GrammarNazi.Core.BotCommands.Telegram
+{
+    public abstract class BaseTelegramCommand
+    {
+        protected readonly ITelegramBotClient _client;
```

Comment:
Use Pascal case naming for this protected field

Reply:
Done @nminaya